miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -45,10 +47,25 @@ abstract class MiniAppMessageBridge {
      * @param content The content property of [ShareInfo] object.
      * @param callback The executed action status should be notified back to mini app.
      **/
-    abstract fun shareContent(
+    open fun shareContent(
```

Comment:
Just a suggestion: how about moving this function a bit later probably before/after permission functions to maintain a consistent sorting of features in this file?

Reply:
Can we use this function as the [USERGUIDE](https://github.com/minh-rakuten/android-miniapp/blob/f9737d9a41b7b52864760325d6f34a1db703ef75/miniapp/USERGUIDE.md) mentioned?

Reply:
Custom permission goes after permission function.
Yes, the user still can override it. MINI-1695 goal is to have default implementation but not prevent the override.